### PR TITLE
Stress bug fix. Call Marshal.Release on WindowFrame and PackageManage…

### PR DESCRIPTION
**Pull request by deepakaravindr**
…rModel

@yishaigalatzer , @feiling

### CODE REVIEW

File: src/VsExtension/NuGetPackage.cs [hunk 1]
```diff
@@ -823,15 +827,16 @@ public void AddWindowSettings(string key, UserSettings obj)
...
             var guidCommandUI = Guid.Empty;
-            var caption = String.Format(
+            var caption = string.Format(
                 CultureInfo.CurrentCulture,
                 Resx.Label_NuGetWindowCaption,
                 solutionName);
             var documentName = _dte.Solution.FullName;
+
+            var ppunkDocView = Marshal.GetIUnknownForObject(windowPane);
+            var ppunkDocData = Marshal.GetIUnknownForObject(model);
```
Comment (yishaigalatzer) [inline src/VsExtension/NuGetPackage.cs]: need try/finally (also above)

**Comment by yishaigalatzer** [comment]
fix the try/finally and merge (move the bug to the right milestone as well)

**Comment by deepakaravindr** [comment]
@feiling

**Comment by feiling** [comment]
Looks good. :shipit:

**Comment by emgarten** [comment]
:shipit:

**Comment by deepakaravindr** [comment]
Merged with
- https://github.com/NuGet/NuGet.VisualStudioExtension/commit/c025aa2bfad7ef52455dda4137c310b121b40511
- https://github.com/NuGet/NuGet.VisualStudioExtension/commit/629356da38add978ef7c5ea5661399228d565b6c

**Pull request closed without merging**